src/components/SignUpPage/SignUpPage.js

```diff
@@ -7,7 +7,7 @@ function SignUpPage(){
     <div>
```

Comment:
Given the similarities to the SignIn page, I recommend merging the two allowing for more reusability and less repetition of code.

Reply:
I created an AuthenticationPage component which takes a param if ever the user chooses sign-up or sign-in, the component will fill the page accordingly.